src/algorithms/tracking/CKFTracking.cc

```diff
@@ -82,11 +81,12 @@
 #include <utility>
 
 #include "ActsGeometryProvider.h"
-#include "DD4hepBField.h"
 #include "extensions/edm4eic/EDM4eicToActs.h"
 #include "extensions/spdlog/SpdlogFormatters.h" // IWYU pragma: keep
 #include "extensions/spdlog/SpdlogToActs.h"
 
+// IWYU pragma: no_include "Acts/Utilities/detail/ContainerIterator.hpp"
```

Comment:
@veprbl This is what I thought would cause the detail headers not to get included, but it still created https://github.com/eic/EICrecon/pull/2244/changes.